front/components/assistant/conversation/ConversationContainer.tsx

```diff
@@ -196,6 +197,7 @@ export function ConversationContainer({
               type: "error",
             });
           }
+          throw conversationRes.error;
```

Comment:
Why are we throwing here?

Reply:
to fail the promise and reset the inputbar as this method is passed down as `handleSubmit` in the inputbar.

adding a comment

Reply:
Technically you're catching an error you throw here :) This is forbidden. Any chance we can make it work with an Result<> ?

Reply:
yes, working on it.

Reply:
THANK YOU \o/